packages/ui-components/src/components/breadcrumb-list/test/breadcrumb-list.e2e.ts

```diff
@@ -0,0 +1,56 @@
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
```

Comment:
I think that since this component is only a wrapper with a slot that doesn't make much sense to test it. 
You should move these tests to `test/breadcrumb.e2e.ts`.